README.md

```diff
@@ -1,295 +1,202 @@
+With a basic idea of basic `bash` commands, we can start to play around. We'll
+show you how to make files and directories and interact with them further. 
 
-### Identify My Logged-In Username
-
-Let's start simply. Let's ask the computer who I am logged in as:
-
-```bash
-$ whoami
-```
-
- < probably need screen shot>
-
-The `whoami` command lets you see which user account you're logged in to from a
-terminal window. This might seem obvious, especially if you're logged in on your
-personal computer, but it's not always clear which account you're you're working
-in the command line.
-
-In a strange circumstance where `whoami` isn't installed, there is another command
-you can use that can tell you your current username. Try:
-
-```bash
-$ id -un
-```
-
-### Identifying the "Present Working Directory" With `pwd`
+### Use `ls` to List Files in Shell
 
-With your Terminal program open, type in `pwd` and hit return/enter.
+To start, let's show our files and folders. In a new terminal, try this:
 
 ```bash
-$ pwd
+$ ls
 ```
 
-You should see some output describing the directory you are currently within.
-The `pwd` command stands for "**p**rint **w**orking **d**irectory".
-You'll see that the operating system created a directory under the "User"
-directory that was named the same as your `whoami` result
-
-`/Users/[your user name]`
-
-We would call this directory your "home" directory.
-
-That output is describing a location on your computer.
-
-`/User/kellyegreene` means that I am currently working within a directory `/Users`
-on the root of my machine, and then within that directory, a directory named
-`kellyegreene`. You might wonder what directory `Users` is in. Read on to
-learn more about the `root` directory which contains all directories....
-
-<something like that>
-
-That's my home directory.
-
-Because it's so common to read files from, or move things into our home
-directories, `bash` lets us type `~` as a shortcut. We'll be working with this
-shortcut later on.
+The command `ls` stands for "**l**i**s**t", so you should see a list of all the
+folder and file names in the current working directory. If you pass a directory
+name with the `ls` command, you can view all the files and folders in that
+directory. 
 
-Try this:
+### Recognize the Use of Flags With Commands
 
-**Note:** *Any time you see the* `$` *character, you shouldn't type it in.
-This is just a standard way to represent a bash prompt. Yours may or may not
-be a* `$`.
+We can use flags on most commands to give more specific instructions or to
+change the way the feedback is presented. Most programs also accept flags, or
+options for execution. A flag is denotated by a `-` ("dash").  A common flag
+that nearly all programs and commands accept is a standalone `h`, for
+"**h**elp".
 
 ```bash
-$ cd ..
-$ pwd
+$ ls -h
 ```
+**Note:** *In some programs, options are passed directly to the command and not via flags.*
```

Comment:
provide example: `hugo help`  tells the program `hugo` to print its help output. There's no guarantee about how a program will present information.